GitUI/CommandsDialogs/FormMergeBranch.cs

```diff
@@ -102,6 +109,21 @@ private void OkClick(object sender, EventArgs e)
             }
         }
 
+        private void WriteMessageToMergeFile(string mergeMessage)
```

Comment:
Please expose it via a `TestAccessor` and add unit tests using `ReferenceRepository`

Reply:
This is the method in FormCommit.cs that I used to write this method.

![image](https://user-images.githubusercontent.com/4055444/75968380-6453e280-5f21-11ea-8ab3-4590d72503f2.png)

This method doesn't seem to have any tests. Neither is any of the invoking code covered by tests via the `TestAccessor`.

I'm afraid I don't have much idea about this test that you're proposing.